app/models/site_setting.rb

```diff
@@ -191,6 +191,15 @@ def self.Upload
     SiteSetting::Upload
   end
 
+  class Whispers
+    def self.allowed_group_ids
+      SiteSetting.whispers_allowed_groups.present? ? SiteSetting.whispers_allowed_groups.split("|").map(&:to_i) : []
+    end
+  end
+  def self.Whispers
+    SiteSetting::Whispers
+  end
```

Comment:
I find `SiteSetting.Whispers` very weird. I know `SiteSetting.Upload` does this but I don't know why we do it. 

`SiteSetting.whispers_allowed_group_ids` is sufficient here IMO